<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>xml2rfc</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets.qrc">
    <normaloff>:/assets/xml2rfc.png</normaloff>:/assets/xml2rfc.png</iconset>
  </property>
  <widget class="QWidget" name="mainFrame">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>8</number>
    </property>
    <item>
     <widget class="QFrame" name="settingsFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>-1</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Input file</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="margin">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="sourceLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>None</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Output</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="margin">
           <number>8</number>
          </property>
          <item>
           <widget class="QCheckBox" name="formatPaged">
            <property name="text">
             <string>Paginated Text</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="formatRaw">
            <property name="text">
             <string>Raw Text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="formatHtml">
            <property name="text">
             <string>HTML</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="formatNroff">
            <property name="text">
             <string>Nroff</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="convertButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Convert</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="documentFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Disable tabwidget in-set bg's */
QTabWidget::pane {background: none;}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="xmlTab">
           <attribute name="title">
            <string>XML</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>1</number>
            </property>
            <item>
             <widget class="LinedEditor" name="textXml">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>2</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QGroupBox" name="consoleGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Console</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextBrowser" name="textConsole">
             <property name="openLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuFormats">
     <property name="title">
      <string>Output formats</string>
     </property>
     <addaction name="actionFormatPaged"/>
     <addaction name="actionFormatRaw"/>
     <addaction name="actionFormatHtml"/>
     <addaction name="actionFormatNroff"/>
    </widget>
    <addaction name="menuFormats"/>
    <addaction name="actionValidation"/>
    <addaction name="actionOptionVerbose"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionInstallCLI"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Xml2rfc</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open Source</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear Queue</string>
   </property>
  </action>
  <action name="actionD">
   <property name="text">
    <string>&gt; df</string>
   </property>
  </action>
  <action name="actionFormatPaged">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Paginated Text</string>
   </property>
  </action>
  <action name="actionFormatRaw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Raw Text</string>
   </property>
  </action>
  <action name="actionFormatHtml">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="actionFormatNroff">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nroff</string>
   </property>
  </action>
  <action name="actionOptionVerbose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Verbose console output</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Source</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionValidation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DTD Validation</string>
   </property>
  </action>
  <action name="actionInstallCLI">
   <property name="text">
    <string>Install command-line utility</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LinedEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>editor</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../assets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionFormatPaged</sender>
   <signal>toggled(bool)</signal>
   <receiver>formatPaged</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFormatRaw</sender>
   <signal>toggled(bool)</signal>
   <receiver>formatRaw</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFormatHtml</sender>
   <signal>toggled(bool)</signal>
   <receiver>formatHtml</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>816</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFormatNroff</sender>
   <signal>toggled(bool)</signal>
   <receiver>formatNroff</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>876</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>formatPaged</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionFormatPaged</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>formatRaw</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionFormatRaw</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>743</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>formatHtml</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionFormatHtml</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>816</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>formatNroff</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionFormatNroff</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>876</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
